network/p2p/p2pconf/gossipsub.go

```diff
@@ -64,6 +64,20 @@ type GossipSubConfig struct {
 
 	// PeerScoring is whether to enable GossipSub peer scoring.
 	PeerScoring bool `mapstructure:"gossipsub-peer-scoring-enabled"`
+
+	SubscriptionProviderConfig SubscriptionProviderParameters `mapstructure:",squash"`
+}
+
+type SubscriptionProviderParameters struct {
```

Comment:
Are there concrete minimum values we can use for the validate tags of this struct? For SubscriptionUpdateInterval perhaps we can use 10m as minimum, and CacheSize the current amount of nodes on the current network.

Reply:
Thanks for your input. Setting a fixed minimum seems risky. Presently, validation mandates parameters to be greater than zero, ensuring they're actively configured. Allowing node operators to adjust parameters to minimal values as necessary seems more prudent.